Userland/Libraries/LibJIT/X86_64/Assembler.h

```diff
@@ -742,6 +742,17 @@ struct X86_64Assembler {
         }
     }
 
+    void increment(Operand op)
+    {
+        if (op.is_register_or_memory()) {
+            emit_rex_for_OI(op, REX_W::Yes);
```

Comment:
`OI` is "Opcode + Immediate" (according to @Hendiadyoin1), and since there's no immediate here, we should probably use `emit_rex_for_slash()` instead :)

Same for the decrement change.